ff/src/lib.rs

```diff
@@ -185,6 +191,7 @@ pub trait PrimeFieldRepr:
     }
 
     /// Reads a little endian integer into this representation.
+    #[cfg(feature = "std")]
     fn read_le<R: Read>(&mut self, mut reader: R) -> io::Result<()> {
```

Comment:
This API will be replaced with `no_std`-compatible `Field::from_bytes` (matching the `bls12_381` and `jubjub` APIs) when we remove `PrimeFieldRepr`.